libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/NotificationDataFactory.kt

```diff
@@ -29,41 +28,35 @@ import io.element.android.libraries.push.impl.notifications.model.SimpleNotifiab
 import io.element.android.services.toolbox.api.strings.StringProvider
 
 interface NotificationDataFactory {
-    suspend fun toNotifications(
-        messages: List<NotifiableMessageEvent>,
-        currentUser: MatrixUser,
+    suspend fun List<NotifiableMessageEvent>.toNotifications(
```

Comment:
This is a bit weird to use extension method on an interface like this

Reply:
be807f4